components/03-modules/cart-list-item/mocks/cartListItem.json

```diff
@@ -0,0 +1,22 @@
+{
+  "productImage": {
+    "src": "data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==",
+    "dataSrc": "/images/product/cart-product-160_160.jpg",
+    "alt": "product name goes here"
+  },
+  "productName": "Some product long name",
+  "productPrice": "$85.00",
+  "productId": "qty1",
+  "label":"Qty:",
+  "placeholder": "Qty",
```

Comment:
`label` and `placeholder` shouldn't be here. We should keep here only data from the backend.  We can add it as a prop and set default value equal `Qty`, What do you think?

Reply:
Good point ;)